src/cli/export/export.go

```diff
@@ -104,13 +104,14 @@ func runExport(
 	// It would be better to give a progressbar than a spinner, but we
 	// have any way of knowing how many files are available as of now.
 	diskWriteComplete := observe.MessageWithCompletion(ctx, "Writing data to disk")
-	defer close(diskWriteComplete)
 
 	err = export.ConsumeExportCollections(ctx, exportLocation, expColl, eo.Errors)
 	if err != nil {
 		return Only(ctx, err)
 	}
 
+	close(diskWriteComplete)
```

Comment:
could you please do 2 things for this:
1. move this up to just after the `err = export.ConsumeExportCollections` so it's between that and the error check. That ensures the progress bar is always closed even if we failed the export
2. please add a comment saying why we need this. It looks like the real reason is that `Infof` and friends [attempt to flush the progress bars](https://github.com/alcionai/corso/blob/8102523dc62c001b301cd2ab4e799f86146ab1a0/src/cli/print/print.go#L151) prior to printing so we don't clobber output